internal/pkg/deploy/cloudformation/stack/worker_svc.go

```diff
@@ -180,3 +185,15 @@ func (s *WorkerService) Parameters() ([]*cloudformation.Parameter, error) {
 func (s *WorkerService) SerializedParameters() (string, error) {
 	return serializeTemplateConfig(s.wkld.parser, s)
 }
+
+func (s *WorkerService) containerPortMappings() ([]*template.PortMapping, error) {
```

Comment:
similar to previous duplication comments, can we think of ways such that workerService and scheduledJob uses the same function?